coding/traffic.hpp

```diff
@@ -0,0 +1,106 @@
+#pragma once
+
+#include "coding/reader.hpp"
+#include "coding/writer.hpp"
+
+#include "std/vector.hpp"
+
+// todo(@m) Endianness-agnostic?
+
+namespace coding
+{
+struct TrafficDataPoint
+{
+  TrafficDataPoint() = default;
+
+  TrafficDataPoint(uint64_t timestamp, double lat, double lon): m_timestamp(timestamp), m_lat(lat), m_lon(lon) {}
+
+  // bool operator<(TrafficDataPoint const & o) { return m_timestamp < o.m_timestamp; }
+  
+  uint64_t m_timestamp = 0;
+  double m_lat = 0.0;
+  double m_lon = 0.0;
+};
+
+static double const minLat = -90;
```

Comment:
Please, get rid of static. Or transform to int (but this is in general is a bad idea).